docs/HACKING.md

```diff
@@ -117,6 +117,40 @@ hack/run-ci-e2e-test.sh -s -m 2 -c 1
 ```
 Please note that you do not need to run `hack/olm.sh run` before `hack/run-ci-e2e-test.sh`.
 
+### Running e2e tests on platform-agnostic infrastructure
```

Comment:
`####` as this is sub section under `Running e2e tests on a cluster`. Cluster includes `platform=none`.